src/utils/youtube.js

```diff
@@ -7,3 +8,30 @@ export function createYoutubeEmojiNode(emote) {
   newNode.setAttribute('data-emoji-id', emote.id);
   return newNode;
 }
+
+export function getLiveChat() {
+  return document.getElementsByTagName('yt-live-chat-renderer')[0]?.__data?.data;
+}
+
+let YoutubeEphemeralMessage = null;
+export async function sendEphemeralMessage(message) {
+  const items = document.querySelector('.yt-live-chat-item-list-renderer > #items');
+  if (items == null) {
+    return;
+  }
+  const visibleItems = getLiveChat()?.visibleItems;
+  if (visibleItems == null) {
+    return;
+  }
+  if (YoutubeEphemeralMessage == null) {
+    YoutubeEphemeralMessage = (await import('../common/components/YoutubeEphemeralMessage.jsx')).default;
+  }
+  const bttvMessageContainer = document.createElement('div');
+  bttvMessageContainer.id = 'bttv-chat-admin-message';
+  const root = ReactDOM.createRoot(bttvMessageContainer);
+  /* eslint-disable-next-line react/jsx-filename-extension, react/react-in-jsx-scope */
+  root.render(<YoutubeEphemeralMessage message={message} />);
+  // Forces youtube to append following messages after the ephemeral message
+  visibleItems.push({bttvMessageRenderer: {}});
```

Comment:
it looks like we have a memory leak from this, no? we are adding items to the list, but who is cleaning them up? additionally, we are making new roots and never cleaning those up?